UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs

```diff
@@ -82,7 +82,7 @@ public struct TypeAndStrength
 			float ProcessingAmount = 0;
 			foreach (var bodyPart in Toloop)
 			{
-				ProcessingAmount += bodyPart.ReagentMetabolism * bodyPart.BloodThroughput * bodyPart.CurrentBloodSaturation * Mathf.Max(0.10f, bodyPart.TotalModified);
+				ProcessingAmount += bodyPart.ReagentMetabolism * bodyPart.BloodThroughput * bodyPart.currentBloodSaturation * Mathf.Max(0.10f, bodyPart.TotalModified);
```

Comment:
why did capitalization change here?

Reply:
Because it was accessing the variable in the reagent Circulated component

Reply:
independet from what object you get it, it should be capitalize the first letter